README.md

```diff
@@ -377,6 +377,7 @@ The following env variables can be used
 | EQ_NEW_RELIC_ENABLED                      | False                 | Enable New Relic monitoring                                                                   |
 | NEW_RELIC_LICENSE_KEY                     |                       | Enable new relic monitoring by supplying a New Relic licence key                              |
 | NEW_RELIC_APP_NAME                        |                       | The name to display for the application in New Relic                                          |
+| COOKIE_SETTINGS_URL                       |                       | URL for the Census Webstie Cookie Settings page                                               |
```

Comment:
This should be `EQ_COOKIE_SETTINGS_URL` to be consistent with the other EQ specific environment variables.